### DISCUSSION TITLE
Link to Vimeo on https://github.com/cucumber

### DISCUSSION THREAD

**Discussion by s2k** [cucumber/common#2180] [Q&A Everything Else] [answered]
Hi all,

On https://github.com/cucumber I tried to follow the link '[Contributions don't have to be code](https://vimeo.com/338591264)' to a video on Vimeo in the section https://github.com/cucumber#get-involved- and got this error on Vimeo (not logged in):

> Sorry, we couldn’t find that page
Make sure you’ve typed the URL correctly, or try searching Vimeo. You could also watch one of the videos below instead.

After logging in the error message changed to:

> Unauthorized
You are unauthorized for this action.

Am I doing it wrong?

**Comment by mpkorstanje** [comment, marked as answer]
I assume the video posted on Vimeo has been deleted by the original author and I assume message would have been comparable to: https://github.com/readme/featured/open-source-non-code-contributions

If you want to contribute, for example by updating the link in the documentation, there should be an edit button somewhere around the top of the page you were reading. :wink:
